contracts/recovery-contracts/RecoveryForwarder.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.4.18;
```

Comment:
why are we using super old solidity version? Newer versions:
- have fixed security issues
- much better optimizations so probably cheaper

Reply:
upgraded